src/OpenColorIO/ops/fixedfunction/FixedFunctionOpData.cpp

```diff
@@ -341,7 +362,52 @@ FixedFunctionOpDataRcPtr FixedFunctionOpData::clone() const
 
 void FixedFunctionOpData::validate() const
 {
-    if(m_style==REC2100_SURROUND_FWD || m_style == REC2100_SURROUND_INV)
+    if(m_style==ACES_GAMUTMAP_13_FWD || m_style == ACES_GAMUTMAP_13_INV)
+    {
+        if (m_params.size() != 7)
+        {
+            std::stringstream ss;
+            ss  << "The style '" << ConvertStyleToString(m_style, true)
+                << "' must have seven parameters but "
+                << m_params.size() << " found.";
+            throw Exception(ss.str().c_str());
+        }
+
+        const double lim_cyan    = m_params[0];
+        const double lim_magenta = m_params[1];
+        const double lim_yellow  = m_params[2];
+        const double thr_cyan    = m_params[3];
+        const double thr_magenta = m_params[4];
+        const double thr_yellow  = m_params[5];
+        const double power       = m_params[6];
+
+        auto check_bounds = [](const std::string & name, double val, double low, double high)
+        {
+            if (val < low || val > high)
+            {
+                std::stringstream ss;
+                ss << "Parameter " << val << " (" << name << ") is outside valid range [" << low << "," << high << "]";
+                throw Exception(ss.str().c_str());
+            }
+        };
+
+        static constexpr double lim_low_bound = 1.0;
```

Comment:
Added a comment as suggested.

The limits come in part from the DCTL implementation from the VWG. I agree these have no real reason to be there, should we extend them to arbitrary high values or just remove them ?

Reply:
If the values are recommended by the Implementation VWG, then we should probably follow their lead.

The comment about lim being clamped to 1.0001 is not currently accurate since the code allows lim==1., but if we clamp it in the constructor, based on my other question, then obviously the comment should stay.

Reply:
Would it make sense to use arbitrary larger bounds for the time being ? I don't believe the current bounds from the DCTL implementation are anywhere near the official recommended settings to use. Could have something like `lim_hi_bound <= 100`, `thr_low_bound >= 0.1`and `pwr_hi_bound <= 10` although the last one might be too much.

Reply:
I have spoken with my co-chairs of the ACES Gamut VWG (@nick-shaw & @carolalynn22) and we concluded that for the FixedFunctionOp we would prefer not to impose any limits here unless there is a numerical or performance reason for it.
Sensible value limits for users of the Parametric Gamut Compression could be set in a UI via slider ranges instead (the VWG has yet to cover those) but we did not think that the underlying op should need to impose limits unless they are needed for numerical or application stability.

So I think the bounds could be something like this? :

`lim_low_bound  = 1.0000001f`
`lim_hi_bound = numeric_limits<float>::max()`

`thr_low_bound = 0.0f` (Where did the current value of 0.4 come from? From the VWG?)
`thr_hi_bound = 0.99999994`

`pwr_low_bound = 1.0`
`pwr_hi_bound  = numeric_limits<float>::max()` (Does this need be lower for numeric stability? Not sure.)

Since the check_bounds() and clamping functions treat the bounds values as inclusive/acceptable values I've chosen the next float32 (IEEE 754) increment / decrement from 1.0 where we really just mean `distance_limit > 1.0` and `threshold < 1.0`.
So if there are concerns about this running on machines with a different, lower precision floating point value representation these in-/decrements could be adjusted accordingly.

I would also think that the values given to the check_bounds() function and the values used for clamping in the op code itself should be identical, right? Otherwise the behavior could end up being counterintuitive.
For instance `FixedFunctionOpData::validate()` would accept a threshold value of 0.9999999 without throwing the parameter exception but the op then quietly clamp it to 0.9999.
(I know it's a bit pedantic but I wanted to mention it anyway...sorry.)

Reply:
Thanks @Tristimulus for the clarification of the VWG position.

The current threshold low bound is from https://github.com/ampas/aces-vwg-gamut-mapping-2020/blob/master/model/GamutCompress.dctl but that was just a placeholder waiting for the group recommendation. 

About the precision concerns, I guess it's hard to say, one potential limiting factor could be OpenGL ES and mobile GPU if we support them, we could lower that to float16 or alternatively just keep the current values ?

Currently the `validate()` allows for threshold and limit == 1 even though they get clamped to different values in the Op code, there was discussion yesterday about that below in the PR, conclusion was to leave it that way but nobody was opposed to your suggested approach, that is everything happening in `validate()` if I understood your point correctly. That might be slightly better to not have hidden clamping happening inside OCIO as well as remove some duplication and risk for errors.

Reply:
Implemented the suggestions from above, only difference is I'm using limits based on float16 instead of float32 but that can be adjusted if wanted.